roles/ospd/overcloud/network/isolation/three-nics-vlans/templates/vlan-internal_api.cfg.j2

```diff
@@ -0,0 +1,7 @@
+DEVICE={{ provisioner.topology.nodes.loadbalancer.network.undercloud.oc_interface }}.101
```

Comment:
Avoid hardcoded values if possible. It looks like these tag values can be taken form the installer.overcloud.network.template.content.parameter_defaults.*VlanID settings.